minvws/nl-kat-coordination
# Give python-docker-boefjes the possibility to use modules that are not part of OpenKAT

### Changes
Python-docker-boefjes' `BoefjeResource` should not get a `module` since the boefje-runner will not execute the boefje in the same context. This would make it that python-docker-boefjes would make OpenKAT crash because OpenKAT will attempt to import the modules the python-docker-boefje requires.

### Issue link
Closes #3582

### Demo
_Please add some proof in the form of screenshots or screen recordings to show (off) new functionality, if there are interesting new features for end-users._

### QA notes
To test the problem, import a non-existing module into a docker-boefje. 
For example inside `boefjes/boefjes/plugins/kat_nmap_tcp/main.py` add the following line at the top of the file
```python
import ihopethispackagedoesnotexist
```
Now when you run a fresh `make kat`, you should notice you are not able to go to the KAT-alogus anymore, and the logs are full of errors.

When doing the same inside this branch this will not be an issue, the issue should only arise when the docker-boefje is being ran.


---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
